homeassistant/components/aemet/config_flow.py

```diff
@@ -0,0 +1,58 @@
+"""Config flow for AEMET OpenData."""
+from aemet_opendata import AEMET
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_API_KEY, CONF_LATITUDE, CONF_LONGITUDE, CONF_NAME
+import homeassistant.helpers.config_validation as cv
+
+from .const import DEFAULT_NAME
+from .const import DOMAIN  # pylint:disable=unused-import
+
+
+class AemetConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Config flow for AEMET OpenData."""
+
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        errors = {}
+
+        if user_input is not None:
+            latitude = user_input[CONF_LATITUDE]
+            longitude = user_input[CONF_LONGITUDE]
+
+            await self.async_set_unique_id(f"{latitude}-{longitude}")
+            self._abort_if_unique_id_configured()
```

Comment:
Please test the abort case when a config entry with the same unique id as is already added to Home Assistant. Use a MockConfigEntry.